src/index.js

```diff
@@ -35,7 +35,7 @@ cosmiconfig('lint-staged', {
                 console.error(err)
             }
 
-            const tasks = generateTasks(config, resolvePaths(files))
+            const tasks = generateTasks(config, resolvePaths(files, sgf.cwd))
```

Comment:
Can we set the `cwd` as a constant in the beginning and reuse? Don't like the coupling with another lib here.